pmm-app/src/pmm-settings/components/Advanced/Advanced.tsx

```diff
@@ -76,75 +68,90 @@ export const Advanced: FC<AdvancedProps> = ({
 
   return (
     <div className={styles.advancedWrapper}>
-      <table>
-        <tbody>
-          <tr>
-            <td>
-              <div
-                className={settingsStyles.labelWrapper}
-                data-qa="advanced-label"
-              >
-                <span>{retentionLabel}</span>
-                <LinkTooltip
-                  tooltipText={retentionTooltip}
-                  link={DATA_RETENTION_URL}
-                  linkText={tooltipLinkText}
-                  icon="info-circle"
+      <Form
+        onSubmit={applyChanges}
+        initialValues={initialValues}
+        render={({
+          values, handleSubmit, valid, pristine
+        }) => (
+          <form onSubmit={handleSubmit}>
+            <table>
```

Comment:
I'm not really in favour of using `table` for the layout. Can we use divs with `display: flex`?

Reply:
Done 👍